README.md

```diff
@@ -1,10 +1,155 @@
+./bruno  This directory has the API to the Seestar in an easy to ues format for the Bruno program or any other program that can send http requests.
+
+./docker This directory has instruction and code to create a docker container that has all the things needed to run the GUI and the device control code
+
+./front This directory has the code to render the Web GUI.  It is named SSC for Simple Seestar Controler. It makes communicating with the Seestar easy.
+
+./raspberry_pi Here you will find instructions and code to allow you to use a raspberry Pi to host the GUI and the device control software.  If you know RPi this make it very easy to build a standalone device.  You won't have to install anyother softwhere just communicate with it with your browser. Look at the README.md in the directory for more info.
+
+./templates This directory contains device driver templates for building a new part of an alpaca driver. It is Python code, you probably won't need this unless you are a very advanced user.
+
+./thunder-test is the start of unit testing of the code with the Thunder product
+
+Along the right side of the page you will see the release information. When you look in a release you will see the source code for the release as well as zips for single directory installation of the project.
+
+
+### Where do I start?
+
+That depends on your goals. (of course)
+
+If you want to get in and learn the code and see how it all works then you can clone this git repository or download the code from the release, or from the main branch.  You will be interested in the ./device and ./front directories at first. 
+
+If you just want to use the GUI and commnicate with your Seestar then the single directory install may be right for you. The standalone install is described below.
+
+## Installation
+
+### Windows/Linux
+The easiest way to install and run on Windows is to download a zip file that will allow you to run from one .exe file
+and everything will come up. If you want to run from source code then you will need to follow the Mac/Source install below.
+
+Download win_seestar_alp.zip from the current release at:
+    <https://github.com/smart-underworld/seestar_alp/releases>
+
+Unzip the file and put the enclosed folder anywhere you would like.
+Open a command window and navigate the the foler that contains seestar_alp.exe
+Now execute seestar.exe
+At first all you will see here is the path to the config.toml, as commands are sent more info will be in the terminal window.
+Once the program prints this path it is ready to go.  If the Seestar is not turned on you will see messages in this command windows.
+
+The web interface should be available in your browser at <http://localhost:5432/>.
+
+If you need to modify the config.toml file it is located in the _internal subdirectory.
+Log files named alpaca.log will be written to the same directory as the executable
+Ctrl-c will close the windows when you are finished
+
+### Mac or Srouce code 
```

Comment:
Typo - Srouce.

Reply:
thanks, I thought i fixed that yesterday!!!